packages/Testing/PHPUnit/Behavior/MovingFilesTrait.php

```diff
@@ -50,13 +50,13 @@ protected function assertFilesWereAdded(array $expectedAddedFileWithContents): v
             $expectedFilePathWithContent = $expectedAddedFileWithContents[$key];
 
             $this->assertSame(
-                $expectedFilePathWithContent->getFilePath(),
-                $addedFilePathWithContent->getFilePath()
+                realpath($expectedFilePathWithContent->getFilePath()),
+                realpath($addedFilePathWithContent->getFilePath())
             );
 
             $this->assertSame(
-                $expectedFilePathWithContent->getFileContent(),
-                $addedFilePathWithContent->getFileContent()
+                $this->normalizeNewlines($expectedFilePathWithContent->getFileContent()),
+                $this->normalizeNewlines($addedFilePathWithContent->getFileContent())
```

Comment:
This might be hell to do for every single content compare we do.
How can we solve it one place without bothering the content?

Reply:
what do you think about the way I scratched togehter in 

https://github.com/rectorphp/rector-src/pull/99/files#diff-42c9f7fb9d75c70447757e107f0b6d551bc2dcf541e91f37192cf2c1ec16fea3R81-R93

Reply:
I like the direction!

What about making it the default behavior of native methods? I mean override phpunit methods

Reply:
> What about making it the default behavior of native methods? I mean override phpunit methods

overriding default behaviour could trigger some WTF moment for drive-by contributors.
I would go with a `PlatformAgnosticAssert` or similar class, so we use this style of assert where required.

I think there will be a lot of places where additional normalization might allow bugs or unwanted behaviour.

Reply:
> WTF moment for drive-by contributors.

What kind of exactly?

I think here its DX of tests. If I have to think about Windows alternative of every assert where might be some dash/space/newline, I will not write tests. The tests should handle this for us without bothering us with extra complexity layer.

Reply:
Ok your decision, I  will give it a try

Reply:
here we go https://github.com/rectorphp/rector-src/pull/120